README.md

```diff
@@ -26,6 +26,26 @@ The `azdev` tool is designed to aid new and experienced developers in contributi
     - For Azure CLI: https://github.com/Azure/azure-cli
     - For Azure CLI Extensions: https://github.com/Azure/azure-cli-extensions
     - Any other repository that you might have access to that contains CLI extensions.
+
+    After forking `azure-cli`, follow the below commands to set up:
+    ```Shell
+    # Clone your forked repository
+    git clone https://github.com/<your-github-name>/azure-cli.git
+
+    cd azure-cli
+    # Add the Azure/azure-cli repository as upstream
+    git remote add upstream https://github.com/Azure/azure-cli.git
+    git fetch upstream
+    # Reset the default dev branch to track dev branch of Azure/azure-cli so you can use it to track the latest azure-cli code.
+    git branch dev --set-upstream-to upstream/dev
+    # Develop with a new branch
+    git checkout -b <feature_branch>
+    ```
+    You can do the same for `azure-cli-extensions` except that the default branch for it is `master`, run `git branch master -u upstream/master` instead.
+
+    To cache your GitHub credentials in Git, you can use a [cache helper](https://docs.github.com/github/using-git/caching-your-github-credentials-in-git). If you or your organization has MFA enabled, please create a [Personal Access Token](https://docs.github.com/github/authenticating-to-github/creating-a-personal-access-token) and use it as the password to login when prompted by the cache helper.
```

Comment:
The latest Git uses https://github.com/microsoft/Git-Credential-Manager-Core which pops a browser automatically if MFA is required:

https://github.com/microsoft/Git-Credential-Manager-Core#current-status

> Two-factor authentication support for GitHub ✓ ✓ ✓

I would suggest we remove this part and only leave the link to https://docs.github.com/en/github/using-git/caching-your-github-credentials-in-git here, as the Git world changes fast everyday.